en/docs/install-and-setup/setup/api-controller/managing-apis-in-kubernetes/managing-apis-in-kubernetes.md

```diff
@@ -0,0 +1,168 @@
+# Managing APIs in Kubernetes
+
+WSO2 API Controller, **apictl** allows you to manage APIs in Kubernetes with the use of WSO2 API operator. This approach uses the kubernetes native way to deploy the APIS into a running microgateway cluster from the command line.
+
+!!! Prerequisites
+
+    - [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+    - [Kubernetes v1.14 or above](https://Kubernetes.io/docs/setup/) <br>
+
+        - Minimum CPU : 2vCPU
+        - Minimum Memory : 2GB
+
+!!! info
+    **Before you begin** 
+
+    -  First download the microgateway distribution from the [github release page](https://github.com/wso2/product-microgateway/releases/) and extract that to a folder of your choice. Then deploy microgateway in kubernetes cluster. Please refer [Steps for deploying  microgateway in kubernetes]()
+
+    -  Make sure the CTL is downloaded and initialized, if not follow the steps in [Download and Initialize the CTL Tool]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-ctl-tool).
```

Comment:
```suggestion
    -  Make sure the apictl is downloaded and initialized, if not follow the steps in [Download and initialize the apictl]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-apictl).
```
Please recheck the links. These have been changed recently.

Reply:
Fixed with 21f466ce5d99225a24a45351eae0997e87f8c2f5